tests/bsim/bluetooth/audio/test_scripts/csip_register.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+VERBOSITY_LEVEL=2
+EXECUTE_TIMEOUT=30
+
+cd ${BSIM_OUT_PATH}/bin
+
+SIMULATION_ID="csip_register"
+
+Execute ./bs_${BOARD_TS}_tests_bsim_bluetooth_audio_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -testid=csip_set_member_register \
+  -RealEncryption=1 -rs=20 -D=1 -argstest rank 1 size 2
+
+# Simulation time should be larger than the WAIT_TIME in common.h
+Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
```

Comment:
Side note: In principle one does not need to connect to the Phy when there is no radio activity in this test.
But I can see the common logic of the test calls the utility function to open the backchannels by default (and that requires the Phy connected).
It is also worth noting that that common logic of the test opens a back channel from the device to itself :)

Reply:
> But I can see the common logic of the test calls the utility function to open the backchannels by default (and that requires the Phy connected).

Yeah, but that could potentially be conditionally done

> It is also worth noting that that common logic of the test opens a back channel from the device to itself :)

Yeah, that is intentional for some of the "send_all" functions. Something that needs to be handled different when we, at some point, move the LE Audio tests to use the testlib